Regression: Could not find a declaration file for module 'qunit-dom' on v1.6.0-beta.1

1. Clone this repo: https://github.com/johanrd/qunit-dom-typescript-reproduction
2. Run `tsc`, and experience the following typescript errors:

```console
machine:qunit-dom-typescript-reproduction user$ tsc
tests/acceptance/application-test.ts:13:12 - error TS2339: Property 'dom' does not exist on type 'Assert'.

13     assert.dom('#ember-welcome-page-id-selector').exists();
              ~~~

tests/test-helper.ts:5:23 - error TS7016: Could not find a declaration file for module 'qunit-dom'. '…/qunit-dom-typescript-reproduction/node_modules/qunit-dom/index.js' implicitly has an 'any' type.
  Try `npm install @types/qunit-dom` if it exists or add a new declaration (.d.ts) file containing `declare module 'qunit-dom';`

5 import { setup } from 'qunit-dom';
``` 

`yarn add @types/qunit-dom` does not make any difference (as expected from deprecation).

Have I missed something from the migration guide and/or readme? Or is it an actual regression on `v1.6.0-beta.1`? 
thanks!

On https://github.com/johanrd/qunit-dom-typescript-reproduction:

```console
$ yarn remove qunit-dom
$ yarn cache clean qunit-dom
$ rm -rf node_modules
$ yarn install
$ yarn add https://github.com/simplabs/qunit-dom#e654d0e1c10fdb2b96f0255775ad66a2c0b24193 --dev
$ tsc

tests/acceptance/application-test.ts:13:12 - error TS2339: Property 'dom' does not exist on type 'Assert'.

13     assert.dom('#ember-welcome-page-id-selector').exists();
              ~~~

tests/test-helper.ts:5:23 - error TS7016: Could not find a declaration file for module 'qunit-dom'. '/Users/johanroed/rendra/qunit-dom-typescript-reproduction/node_modules/qunit-dom/index.js' implicitly has an 'any' type.
  Try `npm install @types/qunit-dom` if it exists or add a new declaration (.d.ts) file containing `declare module 'qunit-dom';`

5 import { setup } from 'qunit-dom';
                        ~~~~~~~~~~~


Found 2 errors.
```

See commit [7ae45ad](https://github.com/johanrd/qunit-dom-typescript-reproduction/commit/88ced1437bc009b13c894a096a9b5cee61e80443#diff-7ae45ad102eab3b6d7e7896acd08c427a9b25b346470d7bc6507b6481575d519R62 ) 
FWIW: VS Code outputs the same errors as `tsc`

--- a/lib/install.ts
+++ b/lib/install.ts
@@ -1,12 +1,6 @@
 import DOMAssertions from './assertions';
 import { getRootElement } from './root-element';
 
-declare global {
-  interface Assert {
-    dom(target?: string | Element | null, rootElement?: Element): DOMAssertions;
-  }
-}
-
 export default function (assert: Assert) {
   assert.dom = function (target?: string | Element | null, rootElement?: Element): DOMAssertions {
     if (!isValidRootElement(rootElement)) {

--- a/lib/qunit-dom-globals.ts
+++ b/lib/qunit-dom-globals.ts
@@ -1,5 +1,0 @@
-/* global QUnit */
-
-import install from './install';
-
-install(QUnit.assert);

--- a/lib/qunit-dom.ts
+++ b/lib/qunit-dom.ts
@@ -1,0 +1,14 @@
+/* global QUnit */
+
+import type DOMAssertions from './assertions';
+import install from './install';
+
+export { setup } from './qunit-dom-modules';
+
+declare global {
+  interface Assert {
+    dom(target?: string | Element | null, rootElement?: Element): DOMAssertions;
+  }
+}
+
+install(QUnit.assert);

--- a/rollup.config.js
+++ b/rollup.config.js
@@ -10,12 +10,13 @@ const typescriptConfiguration = {
 };
 
 const iifeBundle = {
-  input: 'lib/qunit-dom-globals.ts',
+  input: 'lib/qunit-dom.ts',
 
   external: ['qunit'],
   plugins: [typescript(typescriptConfiguration)],
 
   output: {
+    name: 'QUnitDOM',
     file: 'dist/qunit-dom.js',
     format: 'iife',
     sourcemap: true,

